roles/inspec/defaults/main.yml

```diff
@@ -9,9 +9,9 @@ inspec:
   dependency_path: /etc/inspec/
   profiles:
     openstack_security:
-      name: inspec-openstack-security
+      name: openstack-baseline
```

Comment:
You need to update the name to match the profile name which is in https://github.com/dev-sec/openstack-baseline/blob/master/inspec.yml#L1